pandas/tests/base/test_ops.py

```diff
@@ -137,227 +137,233 @@ def setup_method(self, method):
         self.is_valid_objs = self.objs
         self.not_valid_objs = []
 
-    def test_none_comparison(self):
+    def test_none_comparison(self, series_with_differing_index):
+        o = series_with_differing_index
+        if isinstance(o.index, IntervalIndex):
+            pytest.skip("IntervalIndex is immutable")
+        if len(o) < 1:
+            pytest.skip("Test doesn't make sense on empty data")
```

Comment:
does this occur?  this seems like of like `hypothesis`'s `assume(len(o) >= 1)`

Reply:
Yes it does, as we have an [empty in the indices fixture](https://github.com/pandas-dev/pandas/blob/master/pandas/conftest.py#L973)

Reply:
thanks